spec/json/HotelSearch_v2_swagger_specification.json

```diff
@@ -884,10 +884,10 @@
           "description": "check-out date of the stay (hotel local date). Format YYYY-MM-DD\n\nThe lowest accepted value is `checkInDate`+1. \n\n"
         },
         "roomQuantity": {
-          "type": "integer",
-          "minimum": 1,
-          "maximum": 9,
-          "example": 1,
+          "type": "string",
+          "minimum": 1, #to be updated

```

Comment:
minimum and maximum properties to be removed

Reply:
removal is done!